io/src/image_grabber.cpp

```diff
@@ -248,7 +249,6 @@ pcl::ImageGrabberBase::ImageGrabberImpl::ImageGrabberImpl (pcl::ImageGrabberBase
   , principal_point_y_ (239.5)
   , num_threads_ (1)
 {
-  depth_image_files_ = depth_image_files;
   cur_frame_ = 0;
```

Comment:
Why not cur_frame_?

Reply:
I focused on non-trivial types in this commit, where initialization in the initialization list instead of the constructor body has a performance advantage (suggested by cppcheck). `cur_frame_` is of type `std::size_t`, so it should make no difference. But yes, I can also move `cur_frame_ = 0` to the initialization list.